collectd/files/postgresql.conf

```diff
@@ -6,16 +6,66 @@
 # Modify the config that generates this file instead
 #
 
-LoadPlugin postgresql
+<LoadPlugin postgresql>
+</LoadPlugin>
+
+<Plugin postgresql>
+{%- for query in collectd_settings.plugins.postgresql.queries %}
+  <Query {{ query.name }}>
+    {%- if query.statement is defined and query.statement %}
+    Statement "{{ query.statement }}"
+    {%- endif %}    
+    {%- if query.param is defined and query.param %}
+    Param "{{ query.param }}"
+    {%- endif %}
+    {%- for result in query.results %}  
+    <Result>
+      {%- if result.type is defined and result.type %}
+      Type {{ result.type }}
+      {%- endif %}
+      {%- if result.instanceprefix is defined and result.instanceprefix %}
+      InstancePrefix "{{ result.instanceprefix }}"
+      {%- endif %} 
+      {%- if result.instancefrom is defined and result.instancefrom %}
+      InstancesFrom "{{ result.instancefrom }}"
+      {%- endif %} 
+      {%- if result.valuefrom is defined and result.valuefrom %}
+      ValuesFrom "{{ result.valuefrom }}"
```

Comment:
for this parameter `ValuesFrom` and the previous ones `InstancesFrom`, could you please rename the pillars to `instancesfrom` (with S) and `valuesfrom`?